python/ray/serve/_private/replica.py

```diff
@@ -287,6 +285,7 @@ async def __init__(
             cloudpickle.loads(serialized_init_args),
             cloudpickle.loads(serialized_init_kwargs),
             deployment_id=deployment_id,
+            event_loop=self._event_loop,
```

Comment:
You mentioned in the description that you're trying to separate out the event-loops, but this one could be picking up existing one, rather than creating a new one.

BTW, can you help me understand if there's particular issue we're trying to address with that separation or is it just preemptive?